package.json

```diff
@@ -102,7 +102,9 @@
     "glob-parent": "^5.1.2",
     "path-parse": "^1.0.7",
     "ua-parser-js": "^0.7.22",
-    "underscore": "1.13.1"
+    "underscore": "1.13.1",
+    "ansi-regex": "5.0.1",
+    "normalize-url": "4.5.1"
```

Comment:
`"patternfly-react/**/normalize-url": "4.5.1"` is still preferred.

Reply:
Looks like @sgratch just dropped the resolution config for `normalize-url` and just went with the version bump in `yarn.lock`.   That looks ok to me.